source/_integrations/zha.markdown

```diff
@@ -268,11 +268,11 @@ from the same group:
 
 ### Service `zha.remove`
 
-This service removes an existing device from the network.
+This "Zigbee Home Automation: remove" service removes an existing device from the network. IEEE address of the device can be found on the device card of Zigbee devices, (example of IEEE address data parameter format is `00:0d::6f:00:05:7d:2d:34`).
 
 |  Data | Optional | Description |
 | ---- | ---- | ----------- |
-| `ieee` | no | IEEE address of the device to remove 
+| `ieee_address` | no | IEEE address of the device to remove
```

Comment:
Leave this one as is. Services.yaml needs to be updated. `ieee_address` was deprecated toward `ieee`

Reply:
OK